src/themes/default/components/theme/ButtonOutline.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <component
     :is="link ? 'router-link' : 'button'"
-    :to="localizedRoute(link)"
+    :to="link ? localizedRoute(link) : null"
```

Comment:
i believe we shouldn't direct on users such logic; we're returning here the same object, it's not necessary and overcomplication for newcomers.